src/routes/docs/tooling/command-line/installation/+page.markdoc

```diff
@@ -70,10 +123,33 @@ Before you can use the CLI, you need to login to your Appwrite account using
 appwrite login
 ```
 
+Add the `--endpoint` flag if you're using a self-hosted instance of Appwrite. This flag requires you to add the URL string you're using for your self-hosted instance after the `--endpoint` flag.
```

Comment:
Let's also change the heading for this section so it's different from the Getting started above.